modules/argocd-forking-sample-repository.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-argocd.adoc
+
+[id="argocd-forking-sample-repository_{context}"]
+= Forking and cloning the ArgoCD samples repository
```

Comment:
@dgoodwin @kalexand-rh is this module needed? I was under the impression that we won't include any references to this git repo.

If it should be kept, the git repo needs to transfer to a more official GH org.

Reply:
I think we need to have a reference repo as there would be too much directory structure and examples to include otherwise. We will pursue getting an official repo.

Reply:
We will of course have to maintain the official repo.